[2023-05-02 15:31:52,922] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl_date.clear_data_directory manual__2023-05-02T15:30:45.232220+00:00 [queued]>
[2023-05-02 15:31:52,993] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl_date.clear_data_directory manual__2023-05-02T15:30:45.232220+00:00 [queued]>
[2023-05-02 15:31:52,994] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-02 15:31:52,996] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-05-02 15:31:52,997] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-02 15:31:53,090] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): clear_data_directory> on 2023-05-02 15:30:45.232220+00:00
[2023-05-02 15:31:53,100] {standard_task_runner.py:52} INFO - Started process 494 to run task
[2023-05-02 15:31:53,108] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl_date', 'clear_data_directory', 'manual__2023-05-02T15:30:45.232220+00:00', '--job-id', '716', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow_date.py', '--cfg-path', '/tmp/tmpt6cdh5rt', '--error-file', '/tmp/tmp6y01t17_']
[2023-05-02 15:31:53,111] {standard_task_runner.py:80} INFO - Job 716: Subtask clear_data_directory
[2023-05-02 15:31:53,522] {task_command.py:370} INFO - Running <TaskInstance: dag_etl_date.clear_data_directory manual__2023-05-02T15:30:45.232220+00:00 [running]> on host 295d97741f96
[2023-05-02 15:31:54,269] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl_date
AIRFLOW_CTX_TASK_ID=clear_data_directory
AIRFLOW_CTX_EXECUTION_DATE=2023-05-02T15:30:45.232220+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-02T15:30:45.232220+00:00
[2023-05-02 15:31:54,275] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-05-02 15:31:54,277] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm -rf ${pwd}data/* | echo "приехали"']
[2023-05-02 15:31:54,307] {subprocess.py:85} INFO - Output:
[2023-05-02 15:31:54,359] {subprocess.py:92} INFO - приехали
[2023-05-02 15:31:54,389] {subprocess.py:96} INFO - Command exited with return code 0
[2023-05-02 15:31:54,638] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl_date, task_id=clear_data_directory, execution_date=20230502T153045, start_date=20230502T153152, end_date=20230502T153154
[2023-05-02 15:31:54,748] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-02 15:31:55,017] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
